docs/decisions/0003-release-timeline.rst

```diff
@@ -0,0 +1,70 @@
+================
+Release timeline
+================
+
+-------
+Status
+-------
+
+Draft
+
+-------
+Context
+-------
+
+The technical release process is currently documented in `a Confluence page
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release>`__;
+this is not an attempt to replace or duplicate it.  Rather, the BTR group needs
+to document the timeframe of each release, such as:
+
+* When the ``master`` branch should be split into ``open-release/zebrawood.master``.
+* When release candidates come out.
+* What is the deadline for testing.
+* What is the deadline for the release notes to be put up for review.
+* etc
+
+The changes to the current process were initially proposed `in the discussion forum
+<https://discuss.openedx.org/t/a-modest-proposal-for-lilac-and-beyond/4070>`__
+
+---------
+Decisions
+---------
+
+#. Releases will be made twice a year, once on June 9th, and the other on
+   December 9th.
+
+#. The release timeline will proceed as follows, where "zebrawood" is to be
+   replaced with the upcoming release's actual name:
+
+   +---------------------------+------------------------------------------------+
+   | Time prior to release day | Event                                          |
+   +===========================+================================================+
+   | 8 weeks                   | ``open-release/zebrawood.master`` branched off |
```

Comment:
Raised during the BTR Meetup: This timeline means has the branch off release.master is due on the same day that the 3rd tag for the previous release is created. This can create a bit of confusion in terms of deadline and tasks due. Is it worth considering having it set to 7 weeks instead of 8?

Reply:
What we ended up doing is to stagger the tagging process across a couple of days.  It seems to have worked, so I guess we can keep this schedule.

In any case, as discussed in the previous meeting, I removed the "RCs" from the schedule, and replaced them with milestones.